src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/CounterGroup.cs

```diff
@@ -292,7 +303,23 @@ private static void PollForValues()
                         millisecondsTillNextPoll = Math.Max(1, millisecondsTillNextPoll);
                         sleepDurationInMilliseconds = Math.Min(sleepDurationInMilliseconds, millisecondsTillNextPoll);
                     }
+
+                    if (s_needsResetIncrementingPollingCounters.Count > 0)
```

Comment:
Nit: The code as-is looks like it would work, but I think a swap would simplify it a bit:
```suggestion
                    countersToReset = s_needsResetIncrementingPollingCounters;
                    s_needsResetIncrementingPollingCounters = [];
```

Reply:
Thank you! 

I addressed it, just decided to leave ```if (s_needsResetIncrementingPollingCounters.Count > 0)``` there to not create a new list on each iteration
Take a look please ;)